sdks/java/core/src/main/java/org/apache/beam/sdk/util/WindowedValue.java

```diff
@@ -48,7 +49,7 @@
  *
  * @param <T> the type of the value
  */
-public abstract class WindowedValue<T> {
+public abstract class WindowedValue<T>  implements Serializable {
```

Comment:
Yeah, looks like we may have missed another place were data is serialized which we do not use coders to serialize to bytes which causes Spark to serialize it using its own serializer.